# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/gc/GC.h
# Commit: 453aa2faad97
# Full Hash: 453aa2faad97ce68b33bef75cd232e37d759556e
# Author: Jon Coppeard <jcoppeard@mozilla.com>
# Date: 2022-10-29 09:50:10
# Regressor Bug: 1797755
# File Overlap Count: 1
# Description:
#   Bug 1797755 - Part 4: Remove option to set maximum mark stack capacity in release builds r=sfink
#   
#   Currently we have a GC parameter that allows setting a maximum mark stack
#   capacity. This is only ever used by test code, not in the browser. This
#   requires extra unnecessary work in release builds if we move to a different
# ==============================================================================

diff -r d33708e4f8ec -r 453aa2faad97 js/src/gc/GC.h
--- a/js/src/gc/GC.h	Fri Oct 28 15:17:43 2022 +0000
+++ b/js/src/gc/GC.h	Fri Oct 28 15:17:43 2022 +0000
@@ -49,7 +49,6 @@
   _("unusedChunks", JSGC_UNUSED_CHUNKS, false)                             \
   _("totalChunks", JSGC_TOTAL_CHUNKS, false)                               \
   _("sliceTimeBudgetMS", JSGC_SLICE_TIME_BUDGET_MS, true)                  \
-  _("markStackLimit", JSGC_MARK_STACK_LIMIT, true)                         \
   _("highFrequencyTimeLimit", JSGC_HIGH_FREQUENCY_TIME_LIMIT, true)        \
   _("smallHeapSizeMax", JSGC_SMALL_HEAP_SIZE_MAX, true)                    \
   _("largeHeapSizeMin", JSGC_LARGE_HEAP_SIZE_MIN, true)                    \